clang_tools/install.py

```diff
@@ -32,7 +32,7 @@ def is_installed(tool_name: str, version: str) -> Optional[Path]:
     ver_major = version_tuple[0]
     if len(version_tuple) < 3:
         # append minor and patch version numbers if not specified
-        version_tuple += (0,) * (3 - len(version_tuple))
+        version_tuple += ('0',) * (3 - len(version_tuple))
```

Comment:
```suggestion
        version_tuple += ("0",) * (3 - len(version_tuple))
```

Reply:
The black formatter would've fixed this in the pre-commit stage, but it doesn't seem to be used in this repo's pre-commit config.